docs/modules/ROOT/pages/type-definitions/schema-configuration/type-configuration.adoc

```diff
@@ -0,0 +1,234 @@
+
+[source, graphql, indent=0]
+----
+type Movie { 
+    title: String 
+    length: Int
+}
+----
+
+From this type definitions, the library generates the following operation fields:
+
+**Query**:
+
+    * `movies`
+    * `moviesAggregate`
+    * `moviesConnection`
+
+**Mutation**:
+
+    * `createMovies`
+    * `deleteMovies`
+    * `updateMovies`
+
+**Subscription**:
+
+    * `movieCreated`
+    * `movieUpdated`
+    * `movieDeleted`
+.
+
+This section explains how to reduce the operation fields produced using the directives `@query`, `@mutation`, `@subscription`.
+
+== `@query`
+
+This directive is used to limit the availability of query operations in the library.
+
+=== Definition
+
+[source, graphql, indent=0]
+----
+directive @query(read: Boolean! = true, aggregate: Boolean! = true) on OBJECT | SCHEMA
+----
+
+NOTE: Please note that aggregations will no longer be generated by default in the 4.x version of the library. See xref::guides/v4-migration/index.adoc#opt-in-aggregation[`Opt-in Aggregation`]
+
+=== Usage
+
+==== Disable _movies_ and _moviesConnection_ operations
+
+[source, graphql, indent=0]
+----
+type Movie @query(read: false, aggregate: true) { 
+    title: String 
+    length: Int
+}
+----
+
+==== Disable _moviesAggregate_ operations
+
+[source, graphql, indent=0]
+----
+type Movie @query(read: true, aggregate: false)  { 
+    title: String 
+    length: Int
+}
+----
+
+== `@mutation`
+
+This directive is used to limit the availability of Mutation operations in the library.
+
+=== Definition
+
+[source, graphql, indent=0]
+----
+enum MutationFields {
+    CREATE
+    UPDATE
+    DELETE
+}
+
+directive @mutation(operations: [MutationFields!]! = [CREATE, UPDATE, DELETE]) on OBJECT | SCHEMA
+----
+
+=== Usage
+
+==== Disable create, delete, and update operations for _Movie_
+
+[source, graphql, indent=0]
+----
+type Movie @mutation(operations: []) { 
+    title: String 
+    length: Int
+}
+----
+
+==== Enable only Create operations for _Movie_
+
+[source, graphql, indent=0]
+----
+type Movie @mutation(operations: [CREATE]) { 
+    title: String 
+    length: Int
+}
+----
+
+== `@subscription`
+
+This directive is used to limit Subscription operations in the library.
+
+=== Definition
+
+[source, graphql, indent=0]
+----
+enum SubscriptionFields {
+    CREATE
+    UPDATE
+    DELETE
+    CREATE_RELATIONSHIP
+    DELETE_RELATIONSHIP
+}
+
+directive @subscription(operations: [SubscriptionFields!]! = [CREATE, UPDATE, DELETE, CREATE_RELATIONSHIP, DELETE_RELATIONSHIP]) on OBJECT | SCHEMA
+----
+
+=== Usage
+
+==== Disable subscriptions for _Movie_
+
+[source, graphql, indent=0]
+----
+type Movie @subscription(operations: [])  { 
+    title: String 
+    length: Int
+}
+----
+
+==== Enable only movieCreated subscription for Movie
+
+[source, graphql, indent=0]
+----
+type Movie @subscription(operations: [CREATE]) { 
+    title: String 
+    length: Int
+}
+----
+
+== `@exclude` label:deprecated[]
+
```

Comment:
It would be good to have a sentence or two defining what this is before the note about the deprecation

Reply:
Good spot, I reuse the previous description that was mistakenly removed!